src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- Big search -->
+    <form
+      class="search_form"
+      method="get"
+      data-qa="big"
+      action="/">
```

Comment:
apply code styling [recommendations](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) all over the page. Pay attention to point №2
```suggestion
    <form
      class="search_form"
      method="get"
      data-qa="big"
      action="/"
    >
```

Reply:
is not fixed for input tags

Reply:
pay attention here)